extensions/resource-deployment/src/services/tools/azCliTool.ts

```diff
@@ -3,21 +3,24 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { ToolType } from '../../interfaces';
-import * as nls from 'vscode-nls';
+import { EOL } from 'os';
 import { SemVer } from 'semver';
+import * as nls from 'vscode-nls';
+import { Command, OsType, ToolType } from '../../interfaces';
 import { IPlatformService } from '../platformService';
-import { EOL } from 'os';
 import { ToolBase } from './toolBase';
+
 const localize = nls.loadMessageBundle();
+const defaultInstallationRoot = '~/.local/bin';
+const win32InstallationRoot = `${process.env['ProgramFiles(x86)']}\\Microsoft SDKs\\Azure\\CLI2\\wbin`;
```

Comment:
These names don't make sense to me - why is the "default" one linux-based?

Reply:
because we have a special one for win32 and the other one is supposed to work for both darwin and linux. This might change once I am done with my mac and linux testing.